statistics/handle/update.go

```diff
@@ -845,6 +871,30 @@ func (h *Handle) dumpStatsUpdateToKV(tableID, isIndex int64, q *statistics.Query
 	return errors.Trace(err)
 }
 
+// DumpColStatsUsageToKV sweeps the whole list, updates the column stats usage map and dumps it to KV.
+func (h *Handle) DumpColStatsUsageToKV() error {
+	h.sweepList()
+	h.colMap.Lock()
+	colMap := h.colMap.data
+	h.colMap.data = make(colStatsUsageMap)
+	h.colMap.Unlock()
+	defer func() {
+		h.colMap.Lock()
+		h.colMap.data.merge(colMap)
+		h.colMap.Unlock()
+	}()
+	for tblColID, lastUsedAt := range colMap {
+		// TODO: maybe there is a better way to insert UTC timestamp.
+		t := lastUsedAt.UTC().Format(types.TimeFormat)
+		const sql = "INSERT INTO mysql.column_stats_usage (table_id, column_id, last_used_at) VALUES (%?, %?, CONVERT_TZ(%?, '+00:00', @@TIME_ZONE)) ON DUPLICATE KEY UPDATE last_used_at = CASE WHEN last_used_at IS NULL THEN CONVERT_TZ(%?, '+00:00', @@TIME_ZONE) ELSE GREATEST(last_used_at, CONVERT_TZ(%?, '+00:00', @@TIME_ZONE)) END"
+		if _, _, err := h.execRestrictedSQL(context.Background(), sql, tblColID.TableID, tblColID.ColumnID, t, t, t); err != nil {
+			return errors.Trace(err)
+		}
```

Comment:
We can have a test later that we use a batched version
```
insert into ... values (), (), (), .... (), on duplicate key update ...
```
Put 10 or 50 columns at a batch.

Reply:
Done.